stdlib/public/Observation/Sources/Observation/Observable.swift

```diff
@@ -26,6 +51,10 @@
 public macro Observable() =
   #externalMacro(module: "ObservationMacros", type: "ObservableMacro")
 
+/// Synthesizes a property for accessors.
+///
+/// The ``Observation`` framework uses this macro. Its use outside of the
```

Comment:
is framework the right name here? It is a module.

Reply:
Good point. We label Observation as a framework in the documentation, but module is correct.